packages/web/components/cards/bond-card.tsx

```diff
@@ -341,22 +350,24 @@ const IncentiveBreakdownRow: FunctionComponent<
 > = ({ dailyPoolReward, apr, numDaysRemaining }) => {
   const t = useTranslation();
   return (
-    <div className="flex items-start place-content-between">
-      <div className="flex items-center shrink-0 gap-2">
-        <h6 className="text-osmoverse-200">+{apr.maxDecimals(0).toString()}</h6>
+    <div className="flex place-content-between items-start">
+      <div className="flex shrink-0 items-center gap-2">
+        <h6 className="subtitle1 text-white">
```

Comment:
Anywhere there's h6 and a top-level subtitle1 class, let's change it to a `span` element.